### ISSUE TITLE
Copenhagen Elixir and Erlang August Meet Up

### ISSUE THREAD

**Issue by gausby** [cphex/cphex#25]
**Date**: August 12, 2015.
**Time**: 17.00 for the Elixir 101 session, 18.00 for the meetup itself.

**Location**:
E-conomic
Langebrogade 1
1411 København K

**Programme**:
- Elixir 101 session
- Collectable
- Elixir and OTP Design when You Know Nothing

We have suggestions for talks—see below—but we would like to welcome new faces to the stage, so if you have a talk please come forward.

* * *

We need talks—two talks seems to be a good number. Please come forward if you have an idea for a talk, or have a project that you want to present, etc. Ideas so far:

  * Collectable, by Gausby
  * Elixir/OTP Design When You Know Nothing, by @skovsgaard

Talks about Elixir/Erlang for gaming would be great considering Spilhuset who has offered to host the event!

* * *
<strike>We need a date. Any ideas on what part of August would be best? Should we have the meet up in the start, middle, or end of august?</strike> We have decided on the August 12.

<strike>We need a location. We have been around 15 attendants the last couple of meetings—we might be able to do it at e-conomic, but don't be shy if you have a location. [Spilhuset][spilhuset] has offered to host this event..</strike>

I tried to get hold of the venue but they haven't replied yet. No biggie, but we might have to find another venue.

* * *

Please add to this issue, or join us at #cphex on irc.freenode.net if you have ideas, suggestions, and pointers on how to make the this meet up even better.

[spilhuset]: http://spilhuset.com

**Comment by gausby** [comment]
I think it would be neat to have an Elixir 101 talk before the main event. If we say the "official" meet up start at 18h we could have an Elixir 101 session at 17h.

If I can get some material together I would gladly give this session.

This way we could include newcomers to the language, and not scare them away if we do an intermediate or even advanced talk—If no one comes to the session I could just sit in a corner and hack, or help the event host setting up for the event.

**Comment by hanstdam** [comment]
I'd gladly join the 101 session. Great initiative!

**Comment by gausby** [comment]
@hanstdam great. I guess the quality will be "beta" (I am still learning how to communicate this stuff), so I am sure you will be able to give me some good pointers in return :)

**Comment by ahf** [comment]
We'll happily host it again. I'm at CCC around the 13 to 17th of august, so if we are to host it, it has to be very early august or late august :-P

**Comment by gausby** [comment]
@ahf :+1: Let's have eTilbudsavis as a backup so we can mix up the venues a bit :)

**Comment by ahf** [comment]
Agreed!

**Comment by skovsgaard** [comment]
Suggested dates, now that we're getting closer to August: 5., 6., 10., 12.

We could push to late in August, if none of those dates fit. If we need another venue option, I can ask if e-conomic wants to host.

**Comment by gausby** [comment]
I think that August 12 would be good.

**Comment by ahf** [comment]
I'll skip this event, we are already at CCC at that time :( See you guys in September!

**Comment by skovsgaard** [comment]
So are we settled on the 12th already?

**Comment by madsflensted** [comment]
12th works for me

**Comment by gausby** [comment]
Let's decide on the 12th.

Does any of you have ideas for talks? I have been thinking about stupid
ways of sorting lists of numbers; and I could do a pre-event Elixir 101
session for the new guys.

**Comment by skovsgaard** [comment]
If we don't have another venue ready, I'll get something set up with e-conomic today.

**Comment by gausby** [comment]
How about spilhuset as a venue?

  * https://twitter.com/flarup/status/626331213003943936

Would be great if we had some talks about Elixir and Erlang used for gaming :)

**Comment by gausby** [comment]
Let's settle for Spilhuset.

Please come forward if you have an idea for a talk; bonus points if it is about gaming.

I will gladly help you prepare your talk if you are a first time speaker, or think I could help you polish your talk :)

**Comment by skovsgaard** [comment]
If we're still short a talk (and no one comes up with anything gaming related), I can do Elixir/OTP Design When You Know Nothing, based on the very true story of what I'm working on now.

**Comment by gausby** [comment]
Nice, Jonas! I'll add your proposal as an option to the issue then.

**Comment by TheoAndersen** [comment]
Is 19:00 as written on metope.com still the right time for this event?

**Comment by skovsgaard** [comment]
Well, we're still marked TBA on time, but I've got no objections to 1900. Anyone else got a time that'd work better for them or should we just settle on 19?

**Comment by gausby** [comment]
I think I should ask the host when they can have us, and I should probably have an hour with the ones who wants an Elixir 101 before the actual event. In short: Let me get back on this.

**Comment by gausby** [comment]
I tried to contact the venue but they haven't replied yet.

I will put the location needed label back on this issue, as we might have to find another venue.

**Comment by skovsgaard** [comment]
Do we have any options on that front?

**Comment by gausby** [comment]
Some (all perhaps):

  * e-conomic (as you, @skovsgaard, said)
  * eTilbudsavis (if we can get hold of jlouis)

**Comment by skovsgaard** [comment]
Should I check tomorrow?

**Comment by gausby** [comment]
@skovsgaard Please do.

**Comment by skovsgaard** [comment]
We've got a meeting room at e-conomic from 17h tomorrow.

**Comment by gausby** [comment]
Thanks for a great meet up, and thanks to the people who came to the Elixir 101 session. I know it wasn't my best presentation ever (but it wasn't the worst, oh my); but thanks for the questions during the presentation, and I got some great feedback from Mads: I need to find the story that brings it together.

I will work on it and give it again at occasions.

I've put [the code I did for the collectable-show-off session in a Gist](https://gist.github.com/gausby/da2a4a06ae68458fb91e).

…and I hope to see you at all the September meet up. Thanks.
